options/locale/locale_en-US.ini

```diff
@@ -463,6 +463,7 @@ url_error = `'%s' is not a valid URL.`
 include_error = ` must contain substring '%s'.`
 glob_pattern_error = ` glob pattern is invalid: %s.`
 regex_pattern_error = ` regex pattern is invalid: %s.`
+username_error = ` should contain only alphanumeric, dash ('-'), underscore ('_') and dot ('.') characters, and cannot begin or end with non-alphanumeric, consecutive non-alphanumerics are also not allowed.`
```

Comment:
Since the other translations mention explicitly the faulty type, should we perhaps prepend `username`?
```suggestion
username_error = ` username can only contain alphanumeric chars ('0-9','a-z','A-Z'), dash ('-'), underscore ('_') and dot ('.'). It cannot begin or end with non-alphanumeric chars, and consecutive non-alphanumeric chars are also forbidden.`
```

Reply:
I'll take your advice suggestion, but without `username`.

The subject can be Username or Organization Name.

<img width="717" alt="image" src="https://user-images.githubusercontent.com/9418365/199868070-5b6b2547-7220-42bf-884a-a3257fe5fcb7.png">

<img width="814" alt="image" src="https://user-images.githubusercontent.com/9418365/199867946-c8d6af65-e5d4-4bc4-889f-10d38abfe0ce.png">